src/client.ts

```diff
@@ -0,0 +1,32 @@
+
+import axios from "axios"
+import datos from "./datos.json"
+
+
+async function postData() {
+  axios.post('http://95.217.235.69/',datos)
+  .then(function(response){
+    console.log(response);
+  }).catch(function(error){
+    console.log(error);
+  })
+}
+
+
+function getData() {
+  axios.get(`http://95.217.235.69/${process.env.EMAIL}`,{
+    
+    
+    headers:{
+      "Content-Type": "application/json",
+      "x-password": process.env.PASSWORD,
+    }
+  }).then(function(response){
+    console.log(response);
+  }).catch(function(error){
+    console.log(error);
+  })
+}
+
+postData();
+var s = 0
```

Comment:
te sugiero que elimines variables innecesarias y que ejecutes la función de ```getData()``` dentro de este archivo
```suggestion
var s = 0
```